src/libraries/Common/tests/System/Security/Cryptography/AlgorithmImplementations/DSA/DSAKeyFileTests.cs

```diff
@@ -126,6 +127,7 @@ public static void ReadWriteDsa512EncryptedPkcs8()
         }
 
         [Fact]
+        [PlatformSpecific(~TestPlatforms.Android)]
```

Comment:
@bartonjs I've disabled these tests for now since the only supported way to do PBE with RC2 requires a significant implementation split for Android since all of the work needs to be done in Java if we want to support it (instead of doing most of the work in .NET and handling the cipher and the hash separately). Is that ok with you?

Reply:
Yeah.  But rather than make them `PlatformSpecific` I think I'd like to see them become `Conditional` on a new `RC2Factory.IsSupported`, which can certainly start out as

```C#
public static bool IsSupported { get; } = !PlatformDetection.IsAndroid();
```

That way it's more descriptive as to why the test is disabled.

Reply:
I'll change these tests to be conditional on that then.